### DISCUSSION TITLE
想在整个应用的入口做些事件监听，但我找不到类似的钩子去实现

### DISCUSSION THREAD

**Discussion by alindas** [umijs/umi#12868] [Q&A] [answered]
如题，umi 的设计模式是按约定式进行开发，现在我想在应用的入口处做些全局状态监听，比如监听屏幕尺寸变化，但我不能像传统 react 应用那样在入口组件 app.tsx 里进行操作，请问我在 umi 里可以怎么做？

**Comment by xierenyuan** [comment, marked as answer]
可以使用 global.tsx  https://umijs.org/docs/guides/directory-structure#globaljtsx
